### DISCUSSION TITLE
Is there a way to view a thumbnail preview on the seekbar like PotPlayer?

### DISCUSSION THREAD

**Discussion by BluesFanUK** [smplayer-dev/smplayer#963] [Q&A]
About 10 years ago I made the switch from VLC to PotPlayer as it has an incredible option to preview an entire video in real-time just by hovering over the seekbar, other players that have this feature cannot generate the preview quick enough (like it's not quick enough to load the whole reel as you're seeking). SMPlayer doesn't have this feature full stop though, or is it a hidden option/add on somewhere you need to install? Thanks

**Comment by smplayer-dev** [comment]
There no such option yet.

**Reply by IArnaut2** [reply]
When will it be?

**Comment by BluesFanUK** [comment]
Any update yet? Two years on, how is this not standard? PotPlayer has it natively built in, MPC-HC has it but it's clunky. MPV players can have thumbfast added in. Is this never going to be added?

**Comment by smplayer-dev** [comment]
I'm working on it.

**Comment by smplayer-dev** [comment]
Added in this version:
https://github.com/smplayer-dev/smplayer/releases/tag/v10422

**Reply by crazedvole** [reply]
It is not working on mine. I only get a time i.e. 3:22 but no picture

**Comment by tsubasanouta** [comment]
Add the option 'Show thumbnail preview on hover' in preferences -> interface
Where？

**Reply by smplayer-dev** [reply]
> Add the option 'Show thumbnail preview on hover' in preferences -> interface Where？ 

Preferences -> Interface -> Seeking.

**Comment by tsubasanouta** [comment]
The option “Show thumbnail preview on hover” was  included in the latest version.
Thank you so much.
Would you please add  “Show thumbnail preview on hover” to PO files in Transifex?
